r/src/R/ai/h2o/sparkling/H2OContext.R

```diff
@@ -68,6 +68,7 @@ H2OContext.getOrCreate <- function(sc = NULL, conf = NULL) {
     }
     h2o.init(strict_version_check = FALSE, https=https, insecure=insecure, ip = ip, port = port, context_path = contextPath, startH2O = F, username = conf$userName(), password = conf$password())
     setClientConnected(jhc)
+    print(hc)
```

Comment:
Is this a standard way how to log things in R?

Reply:
The purpose of this call is not to log, but really to print.

We do the same in Scala client and Python client to show users on which executors H2O is running, what is the flow UI etc

Reply:
Ok, thanks for your explanation!

Reply:
No worries, better to check :)